.github/workflows/on-release.yaml

```diff
@@ -0,0 +1,16 @@
+name: Tasks on release
+# This top level GH Action Workflow controls what tasks on run when a release is triggered.
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  run-rust-tests:
+    uses: ./.github/workflows/test-rust.yaml
```

Comment:
How come we can't just call ci-rust here?

Reply:
So ci-rust is not a reusable workflow, the test-rust workflow is. That way we can reuse the test-rust workflow everywhere.

This isn't the only configuration, this PR is on the more modular side of the configuration spectrum. This way is kind of a nice touch if you're ever want to add something to the CI process.

Reply:
Hmmm so does this really solve our problem? The release would still be published.

Reply:
No, you didn't indicate you thought that was necessary. I believe, without looking, that we agreed a good test suite was the only reliable way to prevent that from happening. This just makes it easier to reuse those test, it doesn't resolve or close an open issue.

TBH, if huak's builds are failing on release, that's probably a symptom of a bigger problem.

Reply:
This

https://github.com/cnpryer/huak/issues/303#issuecomment-1273593466

Reply:
Gotcha. Yea I think when I imagined a more robust CI it was more so that we were covering the build steps of the publish action.

And I think that's what I'm more familiar with in general. So if we're building to target certain configurations then I'd think we can just throw them into individual actions.

When I'm at my computer and have a second I can link you an example.

Reply:
If you want we can tag team this too. I don't mean to make you do any more work than necessary and this one I think is super important for us.

Reply:
> If you want we can tag team this too. I don't mean to make you do any more work than necessary and this one I think is super important for us.

Sure! I have no qualms with you taking this and running, as long as it helped out.